src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs

```diff
@@ -107,7 +107,7 @@ public TransceiverDirection GetTransceiverDirection(MediaType type)
         /// Sets the transceiver direction for receiving media stream.
         /// </summary>
         /// <remarks>
-        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/> state when transceiver direction is set.
+        /// This method does not throw state exception anymore(Since API12). It can be called in any state.<br/>
         /// </remarks>
         /// <param name="type">The media type.</param>
```

Comment:
consider removing note at line 117 or adding an information that this changed in API 12

Reply:
I'll remove line 117.